migrations/2024-11-15-171347_add_capture_method_sequential_automatic/down.sql

```diff
@@ -0,0 +1,2 @@
+-- This file should undo anything in `up.sql`
+SELECT 1;
```

Comment:
we should remove the enum variant from the type.

Reply:
Will keep it for now